### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by webbdm**
Ready for review         

@zoeames 

        🌮
     🌮🌮🌮
    🌮🌮🌮🌮🌮
   🌮🌮🌮🌮🌮🌮🌮
  🌮🌮🌮🌮🌮🌮🌮🌮🌮
 🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮
🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮

### CODE REVIEW

**Comment by webbdm** [comment]
It looks like taco tree had some issues

**Comment by rousell** [comment]
@webbdm Good job! it looks and works really well. My only suggestion would be to clean up your code - make sure to take out any commented code that isn't used, and check the indentations in your JS file.

File: tree.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    if(keypress.keyCode === 13) { 
+        
+    	
+            if (tree.height === "" || tree.character === "") {
+
+            alert("Can't build yo Tree without both parameters");
+
+    }
+    branchAdder(tree);
```
Comment (rousell) [inline tree.js]: the indentation here is confusing in particular because the ending curly brace for the internal if statement is off.

**Comment by webbdm** [comment]
Fixed :+1:

**Comment by morecallan** [comment]
@webbdm still some small issues in your treeValues function indentation. Come see the indentation queen, @rousell for some clarification. But hint: your  branchAdder function is indented properly.

**Comment by webbdm** [comment]
👍 ?

**Comment by morecallan** [comment]
@webbdm sooo close. Your alert line just needs to be indented:)

**Comment by webbdm** [comment]
OOPS forgot about that space. 

how bout now @morecallan

**Comment by morecallan** [comment]
haha all good now! 👍

**Pull request merged**